lib/routes/tgbus/news.ts

```diff
@@ -0,0 +1,57 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+import { parseArticle } from './utils';
+
+export const route: Route = {
+    path: '/news/',
```

Comment:
```suggestion
    path: '/news',
```
Please remove trailing slash